src/libs/style.ts

```diff
@@ -55,10 +55,6 @@ function indexOfLayer(layers: LayerSpecification[], layerId: string) {
 }
 
 function getAccessToken(sourceName: string, mapStyle: StyleSpecification, opts: {allowFallback?: boolean}) {
-  if(sourceName === "thunderforest_transport" || sourceName === "thunderforest_outdoors") {
```

Comment:
Won't this change break stuff?

Reply:
Oh, this was just moved elsewhere, nevermind then...

Reply:
Yeah sorry I should have flagged that in advance ;) it just moved to avoid all the inspections in multiple places and simply the helper function.